test/Chainweb/Test/Pact/PactSingleChainTest.hs

```diff
@@ -452,6 +457,67 @@ compactionIsIdempotent rdb =
                   putStrLn ""
       assertFailure "pact state check failed"
 
+compactionTransactionIndex :: ()
+  => RocksDb
+  -> TestTree
+compactionTransactionIndex rdb =
+  compactionSetup "compactionTransactionIndex" rdb testPactServiceConfig $ \cr -> do
+    setOneShotMempool cr.mempoolRef goldenMemPool
+
+    let makeTx :: Int -> BlockHeader -> IO ChainwebTransaction
+        makeTx nth bh = buildCwCmd testVersion
+          $ set cbSigners [mkEd25519Signer' sender00 [mkGasCap, mkTransferCap "sender00" "sender01" 1.0]]
+          $ setFromHeader bh
+          $ mkCmd (sshow (nth, bh))
+          $ mkExec' "(coin.transfer \"sender00\" \"sender01\" 1.0)"
+
+    supply <- newIORef @Int 0
+    madeTx <- newIORef @Bool False
+    putStrLn "" -- get around tasty formatting
```

Comment:
Elaborate please

Reply:
the test output without this will look like:

```
  testName: first printed line
second printed line
third printed line
```

but with this change, it will look like:
```
  testName:
first printed line
second printed line
third printed line
```

Reply:
Maybe there's a better way of doing this. I can't recall. Please suggest if you know of one.

Reply:
got rid of the need for this